chall02/vkuikka.c

```diff
@@ -0,0 +1,25 @@
+#include <string.h>
+
+static void		ft_hex_to_str(char *str, int num)
+{
+	char *hex;
+	
+	hex = "0123456789abcdef";
+	str[1] = hex[num % 16];
+	str[0] = hex[num / 16 % 16];
+}
+
+char			*hv_rgb2hex(int r, int g, int b)
+{
+	char	*res;
+
+	if (r < 0 || r > 255 ||
+		g < 0 || g > 255 ||
+		b < 0 || b > 255 ||
+		!(res = strdup("#000000")))
+		return (NULL);
```

Comment:
smart!

I would suggest separate user inputs error cases handling and memory allocation.